MAUI/Cartesian-Charts/Trackball.md

```diff
@@ -258,6 +260,208 @@ The [MarkerSettings](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.
 
 {% endtabs %}
 
+### Trackball Label Template
+
+Trackball labels can be customized by adding any view as its template by using [TrackballLabelTemplate]() of [CartesianSeries](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.CartesianSeries.html).
+
+T> The `DataItem` can be used to access the data linked to the associated business model
```

Comment:
Mention what is binding context for label template.

Reply:
Mentioned the binding context